logging_gocbcore.go

```diff
@@ -0,0 +1,30 @@
+package cbcolumnar
+
+import "github.com/couchbase/gocbcore/v10"
+
+type coreLogWrapper struct {
+	wrapped gocbcore.Logger
+}
+
+func (wrapper coreLogWrapper) Log(level LogLevel, offset int, format string, v ...interface{}) error {
+	return wrapper.wrapped.Log(gocbcore.LogLevel(level), offset+2, format, v...) // nolint:wrapcheck
```

Comment:
Why do we do offset+2 here?

Reply:
I've looked into this before and can't entirely remember now, it's identical to gocb.